src/main/java/com/openlattice/edm/EdmApi.java

```diff
@@ -367,6 +373,13 @@ Void forceRemovePropertyTypeFromEntityType(
     @GET( ENTITY_SETS_BASE_PATH )
     Iterable<EntitySet> getEntitySets();
 
+    @GET( SUMMARY_BASE_PATH )
+    Map<UUID, Stream<PropertySummary>> getAllPropertySummaries( @Body EdmRequest request );
```

Comment:
While it's tempting to use existing EdmRequest object it creates ambiguity in the API as you are just returning Map<UUID, Stream<...>>. If someone passes in UUIDs for other types, I'm guessing nothing happens, so you might want to accept Set<UUID>. If you are factoring in entity types to the property summary then you should have a different return object that makes things more explicit.

Reply:
I changed the implementation so the request doesn't have a body anymore. I'm not sure what you mean about having a more explicit return object.